### PR TITLE
[portal-impl-integration] LPS-95716 CleanUpPortletPreferencesUtilTest

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

File: modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java [hunk 1]
```diff
@@ -122,6 +132,24 @@
 )
 public class EditServerMVCActionCommand extends BaseMVCActionCommand {
 
+	public void cleanUpAddToPagePermissions(ActionRequest actionRequest)
```
Comment (tinatian) [inline modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java:135]: @dantewang 
Why you are creating a new public method here? anyone is using it? If the only usage is to make it easy to invoke in test, we should remove it.

Tina.

**Comment by tinatian** [comment]
@dantewang @ChiZelin 
After inline the util class, the logic of EditServerMVCActionCommand looks complex than before, let do this in two separate tickets to avoid unnecessary risk.
First, move out the util class and integration test for them which is what we want original.
Second, inline the util classes and simplify them, avoid useless methods and so on. You can put this into your list and do it later.

Please send me new pulls.

Thanks.
Tina.

**Comment by tinatian** [comment]
@dantewang 
Sorry Dante, please ignore my last message, I forgot the original reason we inline those methods, we have problems invoke the util class in internal package, right.

I will reopen and look at the pull again.

Tina.

File: modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java (hunk 1 above)
Comment (dantewang) [reply modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java:135]: @tinatian Sorry my bad, I forgot to change the accessibility modifier after moving the method from the `Util` to `EditServerMVCActionCommand`. Will resend.

**Comment by dantewang** [comment]
@tinatian Resend at https://github.com/tinatian/liferay-portal/pull/1404

**Pull request closed without merging**